app/utils/lava_log_parser.py

```diff
@@ -115,7 +160,6 @@ def run(log, boot, txt, html):
         html.write(line)
     html.write("</pre></body></html>\n")
 
-
```

Comment:
Please don't remove these white lines, they are required by `pep8` and make it easier to read the code.

Reply:
I configured a pep8 linter in my editor to avoid this kind of problem in next PRs

Reply:
Nice :)